java/client/src/test/java/glide/api/RedisClusterClientTest.java

```diff
@@ -96,8 +124,134 @@ public TestCommandManager(Response responseToReturn) {
+        // setup
+        CompletableFuture<String> testResponse = mock(CompletableFuture.class);
+        when(testResponse.get()).thenReturn("PONG");
+
+        RequestRoutingConfiguration.Route route = ALL_NODES;
+
+        // match on protobuf request
+        when(commandManager.<String>submitNewCommand(
+                        eq(Ping), eq(new String[0]), eq(Optional.of(route)), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<String> response = service.ping(route);
+        String payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals("PONG", payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void ping_with_message_returns_success() {
+        // setup
+        String message = "RETURN OF THE PONG";
+        String[] arguments = new String[] {message};
+        CompletableFuture<String> testResponse = new CompletableFuture();
+        testResponse.complete(message);
+
+        RequestRoutingConfiguration.Route route = ALL_PRIMARIES;
+
+        // match on protobuf request
+        when(commandManager.<String>submitNewCommand(
+                        eq(Ping), eq(arguments), eq(Optional.of(route)), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<String> response = service.ping(message, route);
+        String pong = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(message, pong);
+    }
+
+    @SneakyThrows
+    @Test
+    public void info_returns_string() {
+        // setup
+        CompletableFuture<ClusterValue<String>> testResponse = mock(CompletableFuture.class);
+        Map<String, String> testPayload = new HashMap<String, String>();
+        testPayload.put("addr1", "value1");
+        testPayload.put("addr2", "value2");
+        testPayload.put("addr3", "value3");
+        when(testResponse.get()).thenReturn(ClusterValue.of(testPayload));
+        when(commandManager.<ClusterValue<String>>submitNewCommand(
+                        eq(Info), eq(new String[0]), eq(Optional.empty()), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<ClusterValue<String>> response = service.info();
+
+        // verify
+        ClusterValue<String> clusterValue = response.get();
+        assertTrue(clusterValue.hasMultiData());
+        Map<String, String> payload = clusterValue.getMultiValue();
+        assertEquals(testPayload, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void info_with_route_returns_string() {
+        // setup
+        CompletableFuture<ClusterValue<String>> testResponse = mock(CompletableFuture.class);
+        Map<String, String> testClusterValue = Map.of("addr1", "addr1 result", "addr2", "addr2 result");
+        RequestRoutingConfiguration.Route route = RequestRoutingConfiguration.SimpleRoute.ALL_NODES;
```

Comment:
If you fix imports you can do
```suggestion
        Route route = SimpleRoute.ALL_NODES;
```

Reply:
Cleaned up.